commands/other/8ball.js

```diff
@@ -0,0 +1,38 @@
+const { Command } = require('discord.js-commando');
+
+module.exports = class EightBallCommand extends Command {
+  constructor(client) {
+    super(client, {
+      name: '8ball',
+      aliases: ['eightball'],
+      memberName: '8Ball',
```

Comment:
i get and errors that memberNames need to be lowercase  suggest memberName: '8ball'

Reply:
yeah you're right I missed that